hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeRecordReader.java

```diff
@@ -251,6 +252,9 @@ private void testReader(boolean partitioned) throws Exception {
           key = recordReader.createKey();
           value = recordReader.createValue();
         }
+        recordReader.getPos();
+        assertEquals(1.0, recordReader.getProgress(), 0.05);
```

Comment:
Maybe add a comment here why the expected value is not strictly equal to 1.0

Reply:
It is strictly equal to 1. It's just that the `assertEquals` function for floats/doubles requires a error margin since floats/doubles are not exact values.